iso.bbx

```diff
@@ -58,34 +64,47 @@
   isbn=true,
   doi=true,
   eprint=true,
-  % sorting=nyt,
-  maxnames=9,
-  minnames=1,
-  citetracker=true,
-  autolang=other,
-  date=year,
-  urldate=iso,
-  seconds=true,
+  maxnames=9,% if possible, all creators should be given in the reference
```

Comment:
My go-to value for *all* is `999`. I guess there are enough papers with more than nine authors for this to be relevant. (We could go for `99`, because `999` would be ridiculous. But `9` seems dangerously low if you really want all names.)

Reply:
The standard says "all names should be given if possible", maybe I was thinking that `9` is reasonable enough because more names are pretty much though (and for more authors print only the first one + et al.), but we can increase to `99`; it still could be overridden as the package option in the user document.